docs/_ext/autoclass_with_examples.py

```diff
@@ -100,14 +94,18 @@ def _create_hint(file_name: str, method_name: str) -> str:
         """
 
 
-def _append_hint(method_name: str, method, base_class: Any, hint: str):
+def _append_hint(method: Any, class_: Any, hint: str) -> None:
```

Comment:
I don't think `-> None` is necessary.

Reply:
I'd rather keep it explicit. It's easier to read later, as it shows that the return type hint wasn't just omitted.